src/foam/nanos/app/AppDownloadBadgeService.js

```diff
@@ -96,6 +97,7 @@ foam.CLASS({
             // Remove the referral token from the URL
             let url = new URL(window.location.href);
             url.searchParams.delete('referral');
+            url.searchParams.delete('utm_id');
```

Comment:
I dont think utm should be deleted, it might need to be used by something else after this service runs

Reply:
Then they will never see the download link. What if they bookmark the link?